core/bug_group_action_api.php

```diff
@@ -115,7 +115,7 @@ function bug_group_action_print_bug_list( array $p_bug_ids_array ) {
 	echo '</table>';
 	echo '</div>';
 
-	if( STATUS_LEGEND_POSITION_BOTTOM == $t_legend_position ) {
+	if( ( $t_legend_position & STATUS_LEGEND_POSITION_BOTTOM ) == STATUS_LEGEND_POSITION_BOTTOM) {
 		echo '<br />';
```

Comment:
This new line is not emitted on the other pages.  I wonder if we should just call into html_top_status_legend() and html_bottom_status_legend().  These would do the check and calling into the html_status_legend().